Source/Archipelago/APRandomizer.cpp

```diff
@@ -449,6 +450,23 @@ std::string APRandomizer::buildUri(std::string& server)
 	return uri;
 }
 
+void APRandomizer::PreGeneration() {
+	// Generate joke hints.
+	std::vector<int> jokeHintOrder(GetJokeHints().size());
+	std::iota(jokeHintOrder.begin(), jokeHintOrder.end(), 0);
+	std::shuffle(jokeHintOrder.begin(), jokeHintOrder.end(), Random::gen);
```

Comment:
Putting this here so there's something to click resolve on (for that dopamine hit)

Random::gen **is** set to a seed before this, but not if the player is doing vanilla puzzles. So right now, vanilla puzzles results in unseeded joke hints

Reply:
My ass really fucked this one up lol my bad

Reply:
Hi